### ISSUE TITLE
GeoLite2 support?

### ISSUE THREAD

**Issue by torataks** [ossec/ossec-hids#1401]
MaxMind stopped updates to its free, legacy GeoIP databases last month.  They recommend that users switch to its free GeoLite2 databases.  Does OSSEC support the newer MaxMind DB format that is used to publish the GeoLite2 info?

https://dev.maxmind.com/geoip/geoip2/geolite2/

Thanks!

**Comment by ddpbsd** [comment]
I don't think we currently support GeoLite2. I haven't looked at it, but I imagine it's a fairly straight forward switch if someone wanted to get their feet wet.

**Comment by atomicturtle** [comment]
updated for 4.4.0
